### PR TITLE
Touch ups

### PR DESCRIPTION
**Pull request by thomwise**
Made a few adjustments to navbar styling and made it function better overall.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -232,7 +232,7 @@ function html5blank_styles()
     wp_enqueue_style( 'bootstrap_css', '//stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css' );
     wp_register_style('style', get_template_directory_uri() . '/style.css', array(), '1.0', 'all');
     wp_enqueue_style('style'); // Enqueue it!
-    // wp_enqueue_style('font-awesome', 'https://cdn.bootcss.com/font-awesome/5.12.1/css/all.min.css'); 
+    wp_enqueue_style('font-awesome', 'https://use.fontawesome.com/releases/v5.13.0/css/all.css');
```
Comment (howdycrystalm) [inline functions.php:235]: Curious why you're not using all.min.css, the min file size will be smaller...

File: sass/pages/resources.scss [hunk 2]
```diff
@@ -13,107 +13,91 @@
...
     font-weight: bold;
     text-shadow: 2px 1px 5px #2c2c2c;
   }
-
   .curves {
     border-radius: 0 0 15px 15px;
   }
-
-  ul>li h5 {
+  ul > li h5 {
```
Comment (howdycrystalm) [inline sass/pages/resources.scss:92]: Good job cleaning up spacing for readability

**Review by howdycrystalm — APPROVED** [review]
I left some comments, for you to checkout. Not using the all.min.css for font-awesome isn't the end of the world, I'm more curious if there was a technical reason you chose not to use it.

Also, we talked about adding the style.css and style.map.css to .gitignore. Go ahead and do that please. You can also delete those two files before pushing up your changes. That's because sass watch will re-generate the files based on style.scss.

File: functions.php (hunk 1 above)
Comment (thomwise) [reply functions.php:235]: I found that cdn on a kind of a sketchy website that had a fair amount of asian symbols on it. It was also causing issues when I would reload the page with my inspector open and would sometimes take forever to load.

File: sass/pages/resources.scss (hunk 2 above)
Comment (thomwise) [reply sass/pages/resources.scss:92]: Thanks!

**Pull request merged**